search/Search_Leaders.dart

```diff
@@ -0,0 +1,20 @@
+void printLeaders(List<int> arr, int size) {
+  for (int i = 0; i < size; i++) {
+    int j;
+    for (j = i + 1; j < size; j++) {
+      if (arr[i] < arr[j]) break;
+    }
+    if (j == size) // the loop didn't break
```

Comment:
We can use a Stack Data Structure to solve this problem which will reduce the time complexity from O(n^2)
to O(N).
in the first pass, we can find the next greater element using stack,
and in next pass we can return the elements in array